.storybook/preview.tsx

```
@@ -15,9 +15,22 @@ const queryClient = new QueryClient({
 
 const preview: Preview = {
   decorators: [
-    (Story) => (
+    (Story, context) => (
       <QueryClientProvider client={queryClient}>
-        <Story />
+        {context.viewMode === 'story' ? (
```

ESLint

[error] Replace `'story'` with `"story"`

(prettier/prettier)